public/index.html

```diff
@@ -12,7 +12,6 @@
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="msapplication-config" content="%PUBLIC_URL%/images/favicons/browserconfig.xml">
     <meta name="theme-color" content="#ffffff">
-    <script src="./env-config.js"></script>
```

Comment:
Does this mean that you no longer need env-config.js?

Reply:
Yes, It's the source of the error because the file does not exist.

Reply:
I have also reached out to Rafal on [Talk](https://talk.openmrs.org/t/developing-the-ocl-for-openmrs-application/17771/806?u=judeatu) for clarification.